graphbolt/include/graphbolt/sampled_subgraph.h

```diff
@@ -0,0 +1,118 @@
+/**
+ *  Copyright (c) 2023 by Contributors
+ * @file graphbolt/sampled_subgraph.h
+ * @brief Header file of sampled sub graph.
+ */
+
+#ifndef GRAPHBOLT_SAMPLED_SUBGRAPH_H_
+#define GRAPHBOLT_SAMPLED_SUBGRAPH_H_
+
+#include <torch/custom_class.h>
+#include <torch/torch.h>
+
+namespace graphbolt {
+namespace sampling {
+
+/**
+ * @brief Struct representing a sampled subgraph.
+ *
+ * Example usage:
+ *
+ * Suppose the subgraph has 3 nodes and 4 edges.
+ * ```
+ * auto indptr = torch::tensor({0, 2, 3, 4}, {torch::kInt64});
+ * auto indices = torch::tensor({55, 101, 3, 3}, {torch::kInt64});
+ * auto reverse_row_node_ids = torch::tensor({3, 3, 101}, {torch::kInt64});
+ *
+ * SampledSubgraph sampledSubgraph(indptr, indices, reverse_row_node_ids);
+ * ```
+ *
+ * The `reverse_row_node_ids` indicates that nodes `[3, 3, 101]` in the
```

Comment:
`[3, 3, 101]` or `[3, 55, 101]`?

Reply:
should be `[3, 3, 101] `,  it is to exaplain same nodes will be mapped to different ids in the row, let me clarify the it.